airbyte-scheduler/src/main/java/io/airbyte/scheduler/SchedulerApp.java

```diff
@@ -93,11 +94,12 @@ public SchedulerApp(Path workspaceRoot,
   public void start() throws IOException {
     final ExecutorService workerThreadPool = Executors.newFixedThreadPool(MAX_WORKERS, THREAD_FACTORY);
     final ScheduledExecutorService scheduledPool = Executors.newSingleThreadScheduledExecutor();
-    final WorkerRunFactory workerRunFactory = new WorkerRunFactory(workspaceRoot, pbf);
+    final SchedulerWorkerRunWithEnvironmentFactory schedulerWorkerRunWithEnvironmentFactory =
```

Comment:
NIT: You can probably shorten the var name to avoid the multiline statement. I think it is still ok to call it workerRunFactory